conda/README.md

```diff
@@ -0,0 +1,30 @@
+# Installation
+
+:::note
+The scripts provided here only work for Linux/macOS.
+
+We need an equivalent bat script for Windows.
+:::
+
+Download and install the `conda` matching your OS: https://docs.conda.io/en/latest/miniconda.html.
+
+Run the following commands:
+
+```bash
+$ conda env create -f conda_env.yaml
+
+$ conda activate taichi-dev
+
+# This will only install the dependencies to the 'taichi-dev' environment.
+(taichi-dev) $ python3 -m pip install --user -r ../requirements_dev.txt
```

Comment:
In this case I think it's safe and cleaner to do `pip install -r ../requirements_dev.txt`?

Reply:
right :-)